openeo/rest/connection.py

```diff
@@ -926,6 +928,17 @@ def job(self, job_id: str):
         """
         return RESTJob(job_id, self)
 
+    def service(self, service_id: str) -> Service:
+        """
+        Get the secondary web service based on the id. The service with the given id should already exist.
+        
+        Use :py:meth:`openeo.rest.connection.Connection.create_service` to create new services
+
+        :param job_id: the service id of an existing secondary web service
+        :return: A service object.
+        """
+        return Service(service_id, self)
```

Comment:
I would write `Service(service_id, connection=self)` here, otherwise the `self` as second argument  looks like a typo (because `self` is usually first argument to a method definition).

Reply:
```suggestion
        return Service(service_id, connection=self)
```